src/components/List.js

```diff
@@ -1,12 +1,31 @@
-import React from 'react';
+import React, { useContext } from 'react';
+import { SearchResultsContext } from '../context/SearchResults';
+import {
+  getLocationsFromResponseAPI,
+  sortLocationsByDistance,
+} from '../helpers/helpersAPI';
+import { Location } from './Location';
 
-const List = () => (
-  <>
-    <h1>List</h1>
-    <div className="view__content">
-      <ul className="list"></ul>
-    </div>
-  </>
-);
+const List = () => {
+  const responseAPI = useContext(SearchResultsContext);
+  const locationsResponse = getLocationsFromResponseAPI(responseAPI);
+  const locations = sortLocationsByDistance(locationsResponse);
```

Comment:
I like how you laid this out into 3 steps and didn’t try to get clever by making it a one-liner!

Reply:
Yes, yes, and these utility function names are fantastic. I understand what the functions are doing by just reading the names.